Cli-CredentialHelper/Installer.cs

```diff
@@ -202,11 +202,11 @@ public void DeployConsole()
             {
                 System.Security.Principal.WindowsIdentity identity = System.Security.Principal.WindowsIdentity.GetCurrent();
                 System.Security.Principal.WindowsPrincipal principal = new System.Security.Principal.WindowsPrincipal(identity);
-                if (!principal.IsInRole(System.Security.Principal.WindowsBuiltInRole.Administrator))
-                {
-                    DeployElevated();
-                    return;
-                }
+                //if (!principal.IsInRole(System.Security.Principal.WindowsBuiltInRole.Administrator))
```

Comment:
Although in the future, might be nice to have it #If !DEBUG-ed so that you can debug into the installation/removal process in VS.

Something for the future